Kickstarter-iOS/Views/Controllers/DiscoveryPageViewController.swift

```diff
@@ -27,12 +27,12 @@ internal final class DiscoveryPageViewController: UITableViewController {
   internal override func viewDidLoad() {
     super.viewDidLoad()
 
+    self.tableView.register(nib: Nib.DiscoveryPostcardCell)
+
     self.tableView.addSubview(self.loadingIndicatorView)
 
     self.tableView.dataSource = self.dataSource
 
-    self.tableView.register(nib: Nib.DiscoveryPostcardCell)
```

Comment:
Ok so this was weird, I was testing and suddenly I started seeing this crash where this table view was attempting to dequeue a cell of type `DiscoverPostcardCell` that hadn't been registered. It was very strange, I hadn't touched any of this work and I kept getting it every time I ran the app. I moved the `register` call to the top (before the `dataSource` is set) and that fixed the crash. No idea why it started happening now out of nowhere, but I figured its a good idea to register the cell before setting the `dataSource` either way.